### DISCUSSION TITLE
Enterprise Scale or Data Management

### DISCUSSION THREAD

**Discussion by zeinab-mk** [Azure/data-management-zone#129] [General]
I was wondering if we could keep Enterprise-Scale Analytics & AI as the official name. Many customers have a good experience with Enterprise Scale Landing Zones in the past which can now help a smoother adoption of the new offering by the customers, it shows consistency and less confusion for internal teams and customers.

Thanks!

Zeinab

**Comment by marvinbuss** [comment]
@zeinab-mk We are currently going through some internal discussions. Stay tuned and I hope we address this issue in a timely manner. I will keep you posted in this thread.

**Reply by arvind-dhariwal-zz** [reply]
@marvinbuss  I am not sure if this confuses to others if we look at repository name - "https://github.com/Azure/data-management-zone" whereas one of the component name of Enterprise Scale Analytics deployment is also  "data-management-zone". 
It would look better if we keep main repo name "enterprise-scale-analytics"

**Reply by marvinbuss** [reply]
Hi @arvind-dhariwal,
So you want us to name this repo "Enterprise-Scale-Analytics" even though it contains the assets for the Data Management Zone?

I agree that this is an option since this repo is the starting point for any user or customer. Let me think about this. 
Do you prefer "Enterprise-Scale Analytics" or "Enterprise-Scale Analytics & AI" or "Enterprise-Scale for Analytics & AI"?

**Reply by arvind-dhariwal-zz** [reply]
Enterprise-Scale Analytics & AI

**Reply by marvinbuss** [reply]
> Enterprise-Scale Analytics & AI

Thanks for your feedback!
